packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2427,6 +2434,9 @@ class _StandardBottomSheetState extends State<_StandardBottomSheet> {
         onClosing: widget.onClosing,
         builder: widget.builder,
         backgroundColor: widget.backgroundColor,
+        elevation: widget.elevation,
```

Comment:
Hmm were these accidentally left out before? If so, can you add a test that catches the bug? If there is no bug then it's possible this never needed to be passed in the first place and you can leave it out.

Reply:
Hmm it actually looks like it's not even possible to hit this code path. Since the `_StandardBottomSheetState` asserts that `animationController` is not null. I think I may simply remove this code path. @HansMuller any idea why this logic was here?